fe1-web/src/features/lao/objects/__tests__/Server.test.ts

```diff
@@ -1,96 +0,0 @@
-import {
-  mockAddress,
-  mockKeyPair,
-  mockLaoId,
-  mockLaoIdHash,
-  mockPopToken,
-  mockPublicKey,
-  mockPublicKey2,
-} from '__tests__/utils';
```

Comment:
Somehow this file was still in the codebase, was replaced by `LaoServer` a while ago.